src/views/reports/base/BaseReport.vue

```diff
@@ -171,25 +188,39 @@ export default {
   }
 }
 
-.no-padding {
-  .content {
-    height: auto;
-    overflow-y: hidden;
-    overflow-x: hidden;
-  }
+  .only-charts {
+    .content {
+      height: auto;
+      overflow-y: hidden;
+      overflow-x: hidden;
 
-  .charts-zone {
-    padding: 0;
-  }
-}
+      .title-bar {
+        margin: 0 30px;
+      }
+
+      .export-btn  {
+        position: absolute;
+        top: 5px;
+        right: 20px;
+      }
+    }
 
-.nav {
-  .content {
     .charts-zone {
-      width: 1040px;
+      // padding: 0;
+    }
+  }
+
+  .nav {
+    .content {
+      .charts-zone {
+        width: 1040px;
+      }
+    }
+
+    .title-bar {
+      width: 100%;
     }
   }
-}
 
 @page {
   size: A4 landscape;
```

Comment:
There are no significant code differences from your provided snippet to analyze for any irregularities, potential issues, or optimizations. This might be due to the complexity of modern web development with many frameworks/libraries being used. If you wish us to review different areas, please specify further.